deployment/hasura/metadata/actions.graphql

```diff
@@ -151,6 +151,7 @@ type ResourceType {
 
 type MerlinSimulationResponse {
   status: MerlinSimulationStatus!
+  simulationDatasetId: Int
```

Comment:
Can this be `Int!`?

Reply:
Good call! I had that in there before I was returning a datasetId in every case

Reply:
Change has been made!